packages/expo-modules-core/CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 
 ### 🐛 Bug fixes
 
+- [iOS] Fix optionals conversion. ([#32239](https://github.com/expo/expo/pull/32239) by [@aleqsio](https://github.com/aleqsio))
```

Comment:
Oops, I think this is in a wrong place now. Can you please move it to unpublished?